variable_selection/fri_kulprit.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>This line looks like it is from the ZIP example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Grupo-de-modelado-probabilista/BART/pull/85/discussion/#comment-1668571922'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yeah, that's the thing this should be a ZIP model, but we have not ZIP support in kulprit, the closest thing is Poisson.</p>